fern/docs.yml

```diff
@@ -332,93 +335,93 @@ navigation:
       - api: API Reference
         paginated: true
 
```

Comment:
Add the SDK references back. Also, can you update CONTRIBUTING.md for generating the SDK docs